evo/tools/plot.py

```diff
@@ -373,7 +374,8 @@ def traj_colormap(ax: plt.Axes, traj: trajectory.PosePath3D,
                     np.amax(traj.positions_xyz[:, 2]))
         if SETTINGS.plot_xyz_realistic:
             set_aspect_equal_3d(ax)
-    fig = plt.gcf()
+    if fig == None:
```

Comment:
Use `is` to check for None, not `== None`.

https://www.python.org/dev/peps/pep-0008/#programming-recommendations

Reply:
Yeah i've been using too much C lately. fixed that too